README.md

```diff
@@ -14,16 +13,15 @@
 
 ## Overview
 
-Imgedit is a package that performs image processing such as resizing and trimming.  
-It's also work on CLI.
+`Imgedit` is a package that performs image processing such as resizing and trimming available on both VLI and GUI.
```

Comment:
This is the first time I have heard the expression `VLI`, is this a common expression?

Reply:
And, this package is not available on GUI.
The GUI part is separated [here](https://github.com/icemint0828/imgedit-wasm).

Reply:
I added an internal link to address that. If you want, we can remove that word altogether.